client/src/fabric/FabricClientConnection.ts

```diff
@@ -12,59 +12,24 @@
  * limitations under the License.
 */
 'use strict';
-import * as fs from 'fs-extra';
-import * as yaml from 'js-yaml';
-
+import { OutputAdapter } from '../logging/OutputAdapter';
 import { IFabricConnection } from './IFabricConnection';
 import { FabricConnection } from './FabricConnection';
-import { OutputAdapter } from '../logging/OutputAdapter';
-
-const ENCODING: string = 'utf8';
+import { FabricWallet } from './FabricWallet';
+import { ExtensionUtil } from '../util/ExtensionUtil';
```

Comment:
is ExtensionUtil used in here?

Reply:
Yeah because I've moved readConnectionProfile